### PR TITLE
feat: Schemas page implemented by schema tree

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/pages/schemas/schemaTree/schemaTree.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  row: NestedData;
+};
+
+const NestedTableRow: React.FC<NestedTableRowProps> = ({ row }) => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      {
+        <>
```
Comment (CptSchnitz) [inline src/pages/schemas/schemaTree/schemaTree.tsx:39]: too many levels, only the <> is needed once

File: src/pages/schemas/schemaTree/testData.ts
Comment (CptSchnitz) [inline src/pages/schemas/schemaTree/testData.ts:1]: is the testData still needed? if not please remove.

**Review by CptSchnitz — CHANGES_REQUESTED** [review]
also let me know what do you think about the suggestions i send you on slack.

File: src/pages/schemas/schemaTree/schemaTree.tsx (hunk 1 above)
Comment (almog8k) [reply src/pages/schemas/schemaTree/schemaTree.tsx]: removed

File: src/pages/schemas/schemaTree/testData.ts
Comment (almog8k) [reply src/pages/schemas/schemaTree/testData.ts:1]: removed

**Review by CptSchnitz — APPROVED** [review]

**Pull request merged**